services/services.rst

```diff
@@ -70,22 +52,36 @@ Public Web Site at pfs.ipmu.jp (http)
 Public project information is at http://pfs.ipmu.jp , 
 such as list of meetings, list of publications, and instrument parameters. 
 
+`PFS blog <http://pfs.ipmu.jp/blog/>`_ is also available on this site. 
+
 PFS project internal website (https pfs.ipmu.jp)
 ------
 
 Every contents at https://pfs.ipmu.jp/ are project only, and you 
-need to log in to view pages. 
-
+need to log in using `PFS LDAP account <account.rst>`_ to view pages. 
 If you have any issue on this site, refer 
 `lost account or change password section <account.rst#lost-account-or-change-password>`_.
 
+With accessing to `PFS project internal website <https://pfs.ipmu.jp/>`_, 
+you will get the newest list of contents in this server as its top page. 
+Short list of frequently used items are:
+
+* `PFS LDAP account manipulator <https://pfs.ipmu.jp/ldap-manip/>`_
+* Content sharing services
+
+  * `Document server <https://pfs.ipmu.jp/docsrv/>`_
+  * `Photo archive <https://pfs.ipmu.jp/photo/>`_
+  * WebDAV storage
```

Comment:
It may be better to note document server is view-only now, although uploading is not limited to PO.
 For instance,  `Document server <https://pfs.ipmu.jp/docsrv/> (view only for most of the members)`

Link to webdav-tech is missing?